components/central-application-gateway/internal/metadata/servicedefservice.go

```diff
@@ -78,7 +80,7 @@ func handleError(err apperrors.AppError, notFoundMessage, internalErrorMEssage s
 	if err.Code() == apperrors.CodeNotFound {
 		return apperrors.NotFound(notFoundMessage)
 	}
-	log.Error(internalErrorMEssage)
+	zap.L().Error(internalErrorMEssage) // TODO: Reconsider this
```

Comment:
What's here to reconsider?

Reply:
Structured logger should have some structured info probably